Assignment_1/palindrome_permutation.js

```diff
@@ -0,0 +1,33 @@
+const palindromePermutation = (str) => {
+
+    const charMap = new Map(); //frequency of each char in string
+    str = str.replace(/\s/g, '').toLowerCase();
+    //remove spaces and convert string to lower case
+  //
+    for (const char of str) {
+        if (charMap.has(char)) {
+          charMap.set(char, charMap.get(char) + 1);
+        } else {
+          charMap.set(char, 1);
+        }
+      }
```

Comment:
Watch your spacing here. Although whitespace doesn't technically matter in JS, it's still easier to read if you're consistent about your spacing.
```suggestion
    for (const char of str) {
      if (charMap.has(char)) {
        charMap.set(char, charMap.get(char) + 1);
      } else {
        charMap.set(char, 1);
      }
    }
```